content/en/agent/kubernetes/_index.md

```diff
@@ -255,6 +254,100 @@ For further details on setting up Operator, including information about using to
+
+Kubernetes resources for Live Containers requires the following set up prior to configuration:
+
+- Agent version >= 7.21.1
+- Process-agent is running (it doesn't need to run the process collection), and configured with the following options:
+
+  {{< code-block lang="yaml">}}
+  - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+    value: "true"
+  - name: DD_ORCHESTRATOR_CLUSTER_ID
+    valueFrom:
+    configMapKeyRef:
+      name: datadog-cluster-id
+      key: id
+  {{< /code-block >}}
+
+  It must also be configured to communicate with the Cluster Agent, see the [Cluster Agent Setup documentation][5] for more information about this.
+
+- Cluster Agent version >= 1.9.0
+- Cluster Agent is running, and the Agent must be able to communicate with it
+
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+If you are using the official [Datadog Helm Chart][1]:
+
+- Use chart version 2.4.5 or above
+- Set `datadog.orchestratorExplorer.enabled` to true in [values.yaml][2]
+- Deploy a new release
+
+
+[1]: https://github.com/DataDog/helm-charts
+[2]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "DaemonSet" %}}
+
+Update the Agent DaemonSet, the Cluster Agent Deployment, and the Cluster Agent ClusterRole with the following settings:
+
+## Cluster Agent Settings
+
+The Cluster Agent container must have the following environment variable configured:
+
+{{< code-block lang="yaml">}}
+- name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+  value: "true"
+{{< /code-block >}}
+
+The Cluster Agent ClusterRole must have the following RBAC permissions:
+
+{{< code-block lang="yaml">}}
+ClusterRole:
+- apiGroups:  # To create the datadog-cluster-id CM
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  - get
+  - update
+...
+- apiGroups:  # Required to get the kube-system namespace UID and generate a cluster ID
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+...
+- apiGroups:  # to collect new resource types
+  - "apps"
+  resources:
+  - deployments
+  - replicasets
+  verbs:
+  - list
+  - get
+  - watch
+{{< /code-block >}}
+
+These permissions are needed in order to create a datadog-cluster-id ConfigMap in the same Namespace as the Agent DaemonSet and the Cluster Agent Deployment, as well as to collect Deployments and ReplicaSets.
+
+If the cluster-id ConfigMap doesn't get created by the Cluster Agent, the Agent pod will not start, and fall in CreateContainerConfigError status. If the Agent pod is stuck because this ConfigMap doesn't exist, update the Cluster Agent permissions and restart its pods to let it create the ConfigMap and the Agent pod will recover automatically.
+
+{{% /tab %}}
+{{% tab "Operator" %}}
```

Comment:
We probably won't have this ready by the time we're going GA, so we should remove it for now and release without it imho. We can add it when the change in the operator is merged and the next operator version is released.